docs/mockups/background.html

```diff
@@ -1647,7 +1656,7 @@ <h4 class="modal-title">
     </span>
    </div>
   </footer>
-  <script language="JavaScript">
+  <script lang="JavaScript">
```

Comment:
I thought the lang attribute was for specifying the human language of the content, not the type of script language. I would expect a type="text/javascript" or nothing here. What am I missing?

Reply:
Not sure about that, but `language` has been deprecated. All browsers (supposedly) use `text/javascript` as the default `type` so that can and is omitted. I think we can remove the newly added `lang` attribute too since it really adds no value.

Reply:
I knew that language was deprecated. I didn't think you could use lang like that in this tag, but I do remember that having nothing there will default to type="text/javascript," so that sounds good.